packages/docusaurus/src/commands/build.ts

```diff
@@ -48,7 +48,7 @@ export default async function build(
     'client-manifest.json',
   );
   let clientConfig: Configuration = merge(
-    createClientConfig(props, cliOptions.minify),
+    createClientConfig(props, cliOptions.minify, cliOptions.useOldCssMinifier),
```

Comment:
I'd rather pass all the options as a single object instead of 1 attribute per option, as it would be more extensible if we need to add other options in the future